### PR TITLE
[ADD] Module which allow to deactivate partner bank accounts

### PR DESCRIPTION
**Pull request by benwillig**

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Review by sbidoul — APPROVED** [review]

File: partner_bank_active/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+===================
+Partner Bank Active
+===================
+
+Allow to deactivate a partner bank account
+
+Installation
+============
+
+Just install it.
```
Comment (yajo) [inline partner_bank_active/README.rst]: Drop installation section then.

File: partner_bank_active/README.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Configuration
+=============
+
+No configuration needed.
```
Comment (yajo) [inline partner_bank_active/README.rst]: Drop configuration section then.

File: partner_bank_active/README.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+Usage
+=====
+
+To use this module, you need to:
+
+. Go on a partner bank account form view
+. Edit the bank account
+. Set it as active/inactive
```
Comment (yajo) [inline partner_bank_active/README.rst]: Use `#.` at the start of lines to get autonumbered lists.

File: partner_bank_active/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner Bank Active',
+    'summary': """
+        Allow to deactivate a partner bank account""",
```
Comment (yajo) [inline partner_bank_active/__manifest__.py]: Single line please

File: partner_bank_active/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/account-financial-tools',
```
Comment (yajo) [inline partner_bank_active/__manifest__.py]: Either PR is in the wrong place, or this line is wrong.

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by benwillig** [comment]
@Yajo Updated this PR based on your review.

**Comment by benwillig** [comment]
Can you update your review @Yajo ? Thanks

**Review by yajo — APPROVED** [review]

**Pull request merged**